content/reference/endpoints/los_records.md

```diff
@@ -0,0 +1,68 @@
+# Los Records
+
+1. TOC
+{:toc}
+
+### Parameters
+<ul class="nav nav-pills" role="tablist">
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+Name                 | Type    | Read/Write | Description
+---------------------|---------|------------|------------
+id                   | Integer | Read       | Los Record's id.
+---------------------|---------|------------|------------
+day                  | [Date](/reference/enums#formats) | Read       | Day for which Los Record is valid.
+kind                 | String  | Read       | Los Record's kind, either `initial` or `final`.
```

Comment:
I think it would be best to extract in enum so we can better describe each kind.